### ISSUE TITLE
Can't py.init a nested class

### ISSUE THREAD

**Issue by gatesn** [spiraldb/ziggy-pydust#167]
Currently we try to import the class from its containing module, but we fail to navigate a hierarchy of nested classes

**Comment by robert3005** [comment]
Related #45?

**Comment by gatesn** [comment]
Yep, similar. The other one I think can be fixed by calling eagerEval on the class / module definition inside `py.class`

**Comment by robert3005** [comment]
The issue there was about instantiating the class. Right now we always check the module for class and never anything else
